internal/api/v1/stage.go

```diff
@@ -153,7 +158,13 @@ func (hc ApplicationsController) Stage(w http.ResponseWriter, r *http.Request) A
 
 	log.Info("staged app", "org", org, "app", params.AppRef, "uid", uid)
 
-	resp := models.StageResponse{Stage: models.NewStage(uid)}
+	if viper.GetBool("use-internal-registry-node-port") {
```

Comment:
Can we add a comment here? E.g. 

```
The ImageURL in the response should be the one accessible by kubernetes. In stageParam above, the registry is passed with the "internal" url, since it's where tekton will push from within the cluster.
```

Reply:
Done.